doc/lua_api.txt

```diff
@@ -5973,10 +5973,11 @@ object you are working with still exists.
 * `get_attach()`: returns parent, bone, position, rotation or nil if it isn't
   attached.
 * `set_detach()`
-* `set_bone_position(bone, position, rotation)`
+* `set_bone_position(bone, position, rotation, stop_animations)`
     * `bone`: string
     * `position`: `{x=num, y=num, z=num}` (relative)
     * `rotation`: `{x=num, y=num, z=num}`
+    * `stop_animations`: stop model animation (optional) default:true
```

Comment:
Oh, so the point is to stop _all_ animation? Why not just make a separate function for disabling the animator then? This doesn't belong in set_bone_position, because again, it's a _set bone position_ function, operating on a bone.